src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -158,4 +158,22 @@ public function getFlashes(string|array $types = null): array
 
         return $result;
     }
+
+    public function getCurrent_Route(): ?string
+    {
+        if (!isset($this->requestStack)) {
+            throw new \RuntimeException('The "app.current_route" variable is not available.');
+        }
+
+        return $this->getRequest()?->attributes->get('_route');
+    }
+
+    public function getCurrent_Route_Parameters(): ?array
```

Comment:
I would add a `@return array<string, mixed>|null` in phpdoc to be more precise about the return type

Reply:
Done!